omnigibson/object_states/inside.py

```diff
@@ -1,8 +1,9 @@
 import torch as th
 
 import omnigibson as og
+from omnigibson.macros import macros
 from omnigibson.object_states.aabb import AABB
-from omnigibson.object_states.adjacency import HorizontalAdjacency, VerticalAdjacency, flatten_planes
+from omnigibson.object_states.adjacency import HorizontalAdjacency, VerticalAdjacency
```

Comment:
Why is this not complaining about unnecessary imports, can you check if they are used somewhere and remove them?

Reply:
oooh great point, forgot to remove them